common/src/main/scala/org/apache/celeborn/common/CelebornConf.scala

```diff
@@ -1988,6 +2006,24 @@ object CelebornConf extends Logging {
       .doubleConf
       .createWithDefault(0.1)
 
+  val SLOTS_ASSIGN_LOADAWARE_FLUSHTIME_WEIGHT: ConfigEntry[Double] =
+    buildConf("celeborn.slots.assign.loadAware.flushTimeWeight")
+      .categories("master")
+      .doc(
+        "Weight of average flush time when calculating ordering in load-aware assignment strategy")
+      .version("0.2.1")
+      .doubleConf
+      .createWithDefault(0)
```

Comment:
Here can be a range validation between 0 and 1.

Reply:
In fact this can be arbitrary value